test/intl402/DisplayNames/prototype/of/type-region-invalid.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-Intl.DisplayNames.prototype.of
+description: Throws a RangeError for invalid `region` codes
+info: |
```

Comment:
Is https://unicode.org/reports/tr35/#unicode-language-identifier the relevant definition? If so, please add the link and copy the EBNF in here as well

Reply:
updated!